javascript/js-in-the-real-world/forms.md

```diff
@@ -23,11 +25,11 @@ Go back to your 'Library' project and add simple validation to that form! Don't
 
 Build a browser form which collects Email, Country, Zip Code, Password and Password Confirmation fields.  It should use live inline validation to inform the user whether a field is properly filled in or not.  That means highlighting a field red and providing a helpful error message until it has been filled in properly.
 
-The form doesn't need to actually submit, but you should give an error message if the button is pushed with any active errors or unfilled required fields.  If all is well and the form is "submitted", give the user a high five.
+The form doesn't need to actually submit, but you should give an error message if the button is pushed with any active errors or unfilled required fields. For the sake of this lesson, make sure **all** of the validation occurs in the JavaScript file. If all is well and the form is "submitted", give the user a high five.
 
 1. Set up a blank HTML document
 2. Think about how you would set up the different form elements and their accompanying validators.  What objects and functions will you need? A few minutes of thought can save you from wasting an hour of coding.  The best thing you can do is whiteboard the entire solution before even touching the computer.
-3. Write the simple form elements.  Don't worry about styling them.
+3. Write the simple form elements. Don't worry about styling them.
```

Comment:
Now that we have some HTML/CSS lessons before this JS lesson, I think this instruction is now out of date and users *should* worry about styling, esepcially since they would have learned about some validation styling by this point (or if they haven't gone back, adding a suggestion that they should complete the HTML/CSS Form Validation lesson first would be good).

Reply:
I've added a new instruction for styling